pkg/plugin/api/client.go

```diff
@@ -15,14 +15,15 @@ import (
 
 	"github.com/vmware-tanzu/octant/internal/log"
 	"github.com/vmware-tanzu/octant/pkg/action"
-	"github.com/vmware-tanzu/octant/pkg/config"
 	"github.com/vmware-tanzu/octant/pkg/plugin/api/proto"
 	"github.com/vmware-tanzu/octant/pkg/store"
 )
 
 //go:generate mockgen -destination=./fake/mock_dashboard_client.go -package=fake github.com/vmware-tanzu/octant/pkg/plugin/api/proto DashboardClient
 //go:generate mockgen -destination=./fake/mock_dashboard_connection.go -package=fake github.com/vmware-tanzu/octant/pkg/plugin/api DashboardConnection
 
+const MaxMessageSize int = 1024 * 1024 * 16
```

Comment:
Can `pkg/config/config.go` be removed since this is done to avoid an import cycle and import `"github.com/vmware-tanzu/octant/pkg/plugin/api"` out of `commands/dash.go`?